server.js

```diff
@@ -1 +1 @@
-console.log('Server')
\ No newline at end of file
+console.log('Server + Test')
```

Comment:
The + test doesn't make sense:
https://docs.gitlab.com/ee/topics/git/unstage.html